measures/measures-schema.yaml

```diff
@@ -132,12 +132,12 @@ definitions:
         description: Performance options mirror numerator options in QCDR measure specifications. Each option comprises a set of codes used to identify instances of performance met, performance not met, performance exclusion or performance exception.
         type: array
         items: { $ref: #/definitions/performanceOption }
+      isRegistryMeasure:
+        type: boolean
```

Comment:
OK! Good idea I'll make it a required field

Reply:
@marimiyachi is that something we need to run by anyone else, since we'll need to add isRegistryMeasure to all the quality measures in measures-data.json and other teams might not be expecting that? or it's ok to just change that ourselves?

Reply:
In this case, adding the field won't break anyone's implementation so it's ok to add ourselves.

Reply:
ok great, added